### PR TITLE
Search User

### PR DESCRIPTION
**Pull request by dungvuit**

### CODE REVIEW

File: lib_manager/app/controllers/admins/users_controller.rb [hunk 1]
```diff
@@ -5,10 +5,23 @@ class Admins::UsersController < ApplicationController
   before_action :find_user, except: [:index, :new, :create]
 
   def index
-    @users = User.sort_by_create_at.paginate page: params[:page]
-    respond_to do |format|
+    if params[:search].present? && params[:search_by_permision].present?
+      @users = User.search_by_name(params[:search])
```
Comment (yeunuthongminh) [inline lib_manager/app/controllers/admins/users_controller.rb]: @users = if if params[:search].present? && params[:search_by_permision].present? 
    User.search_by_name params[:search]

File: lib_manager/app/views/admins/users/index.html.erb [hunk 2]
```diff
@@ -23,6 +23,20 @@
       </div>
       <div class="clearfix"></div>
     </div>
+    <div class="row wrap-search">
+      <div class="col-md-6 col-md-offset-6">
+        <%= form_tag admins_users_path, :method => "get" do %>
```
Comment (yeunuthongminh) [inline lib_manager/app/views/admins/users/index.html.erb]: method: :get

File: lib_manager/app/views/admins/users/index.js.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+$('.table table-bordered table-hover').html('<%= j render "list_user" %>');
```
Comment (yeunuthongminh) [inline lib_manager/app/views/admins/users/index.js.erb]: end file

File: lib_manager/app/models/user.rb [hunk 4]
```diff
@@ -3,6 +3,8 @@ class User < ApplicationRecord
   before_save :downcase_email
 
   scope :sort_by_create_at, -> {order created_at: :desc}
+  scope :search_by_name, -> search{where("name LIKE ?", "%#{search}%")}
```
Comment (hellovietnam93) [inline lib_manager/app/models/user.rb]: Xoá ()

File: lib_manager/app/views/admins/users/index.js.erb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+$('.table').html('<%= j render "list_user" %>');
+$('.pagination').html('<%= j will_paginate @users %>');
```
Comment (yeunuthongminh) [inline lib_manager/app/views/admins/users/index.js.erb]: end file

**Pull request merged**